assets/js/components/SaptuneDetails/SaptuneDetails.test.jsx

```diff
@@ -110,4 +110,54 @@ describe('SaptuneDetails', () => {
       staging.solutions_ids
     );
   });
+
+  it.each([
+    { solution: null },
+    { solution: { notes: [] } },
+    { solution: undefined },
+  ])(
+    'should display dashes for "Enabled Solution," "Applied Solution," "Staged Notes," and "Staged Solutions" when the solution is null, and staging is disabled',
```

Comment:
Too long I would say. `should display empty values when no data is coming` would suffice in my opinion